### PR TITLE
Add some "nothing here" when lists are empty (issue #823)

### PR DESCRIPTION
**Pull request by np**
Fixes: #823

### CODE REVIEW

**Comment by andorp** [comment]
@fisx Please translate the new texts.

File: src/Frontend/Core.hs [hunk 1]
```diff
@@ -336,6 +337,26 @@ percentVotes idea numVoters vv = {- assert c -} v
...
+callToActionOnList
+    :: (Foldable t, Monad m)
+    => HtmlT m ()
+    -> (HtmlT m () -> HtmlT m ())
+    -> (a -> HtmlT m ())
+    -> t a
+    -> HtmlT m ()
+callToActionOnList onCta wrapItems onItem values
+  | null values = div_ [class_ "container-not-found"] $ onCta
+  | otherwise   = wrapItems $ mapM_ onItem values
```
Comment (fisx) [inline src/Frontend/Core.hs]: I've grown to like infixing `mapM_`, but I don't insist yet on adding it to the style guide.  (-:

File: src/Frontend/Fragment/DelegationTab.hs [hunk 2]
```diff
@@ -17,10 +18,33 @@ import Persistent
...
+        renderLi
+        ds
   where
+    delegationLink = case whatsPage of
+        TopicDelegationPage _ctx topic -> topic ^. _Id . to DScopeTopicId . to U.createDelegation
+        UserDelegationPage  _ctx       -> U.createDelegation $ DScopeIdeaSpace SchoolSpace
+
+    showScope = case whatsPage of
+        TopicDelegationPage _ctx _topic -> False
+        UserDelegationPage  _ctx        -> True
```
Comment (fisx) [inline src/Frontend/Fragment/DelegationTab.hs]: I would have said `has _UserDelegationPage` here, but ok.

Comment (andorp) [inline src/Frontend/Fragment/DelegationTab.hs]: I like your idea, I am going to change this.

Comment (andorp) [inline src/Frontend/Fragment/DelegationTab.hs]: I don't want to introduce lens here for the time being.

**Comment by fisx** [comment]
LGTM!

Please check that i didn't break anything and merge if travis is green.

**Pull request merged**